tjdashboard/code/administrator/tables/dashboards.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_TjDashboard
+ *
+ * @author      Techjoomla <contact@techjoomla.com>
+ * @copyright   2017 Techjoomla
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+/**
+ * Tj-Dashboard dashboard table class
+ *
+ * @since  1.0
+ */
+class TjdashboardTableDashboards extends JTable
+{
+	/**
+	 * Constructor
+	 *
+	 * @param   JDatabaseDriver  &$db  Database object
+	 *
+	 * @since  1.0
+	 */
+	public function __construct(&$db)
+	{
+		parent::__construct('#__tj_dashboards', 'dashboard_id', $db);
+	}
+
+	/**
+	 * Overloaded check function
+	 *
+	 * @return  check
+	 *
+	 * @since  1.0.0
+	 */
+	public function check()
+	{
+		$db = JFactory::getDbo();
+
+		// If there is an ordering column and this is a new row then get the next ordering value
+		if (property_exists($this, 'ordering') && $this->dashboard_id == 0)
+		{
+			$this->ordering = self::getNextOrder();
+		}
+
+		if ($this->dashboard_id == 0)
+		{
+			$this->created_by = JFactory::getUser()->id;
+			$this->created_on = JFactory::getDate("now", "UTC")->tosql();
+		}
+
+		$this->modified_by = JFactory::getUser()->id;
+		$this->modified_on = JFactory::getDate("now", "UTC")->tosql();
+
+		$this->alias = trim($this->alias);
+
+		if (empty($this->alias))
+		{
+			$this->alias = $this->title;
```

Comment:
this is wrong. you should sluggify the title. Refer shika code.

Reply:
@coolbung Done in the code below.Checked for Shika code as well they have done same

Reply:
OK